README.md

```diff
@@ -94,58 +94,242 @@ git fetch upstream-code main
 4. Fix the conflicts or other errors. **Note**, that`./rebase.sh` script also apllies the patches from the [`.rebase`](https://github.com/che-incubator/che-code/tree/main/.rebase) directory. Sometimes, it also requires some updates there.
 5. Open a PR with your changes.
 
-## Whitelabel/branding
- 
-There is a functionality to apply branding to some UI elements. The original data can be overridden by executing the [branding.sh](https://github.com/che-incubator/che-code/blob/main/branding/branding.sh) script. The script searches for the branding resources in the [branding folder](https://github.com/che-incubator/che-code/tree/main/branding). 
+## Branding the UI
 
-The `branding/product.json` file is crucial. 
-The values defined in the `branding/product.json` file override the [default values](https://github.com/che-incubator/che-code/blob/main/code/product.json). 
-Also the file serves data mapping for provided branding resources.
+   <details>
+          <summary>You can brand some of the UI elements in the Visual Studio Code - Open Source IDE with your corporate or product brand.</summary>
 
-Example of the `branding/product.json` file:
+\
+This means first adding brand-related files to the forked IDE repository, then building a container image of the branded IDE, and finally adding a `che-editor.yaml` file to the project repository.
+
+Here are some examples of the UI elements in Visual Studio Code - Open Source that you can brand:
+
+* Browser tab title and icon
+* The icon for the empty editor area when no editor is open
+* The Status Bar commands
+* The Status Bar icon
+* The Get Started page
+* The tab icon for the Get Started page
+* The application name in the **About** dialog
```

Comment:
I think it makes sense:
- remove bold style for the `About`
or 
- use bold text for other items above (Status Bar, Get Started)

wdyt?

Reply:
Thank you for spotting these, Roman! I've added bold to all of them.